storage/src/main/java/tech/pegasys/artemis/storage/Store.java

```diff
@@ -104,11 +106,11 @@ public static Store get_genesis_store(final BeaconState genesisState) {
         latest_messages);
   }
 
-  Transaction startTransaction(final StorageUpdateChannel storageUpdateChannel) {
+  public Transaction startTransaction(final StorageUpdateChannel storageUpdateChannel) {
     return startTransaction(storageUpdateChannel, StoreUpdateHandler.NOOP);
   }
 
-  Transaction startTransaction(
+  public Transaction startTransaction(
       final StorageUpdateChannel storageUpdateChannel, final StoreUpdateHandler updateHandler) {
     return new Transaction(storageUpdateChannel, updateHandler);
   }
```

Comment:
Unfortunate that we had to make these public but I think I like having client and server packages not depend on each other enough that it's the right trade off.